### ISSUE TITLE
Absolute positioning on RTL websites reverse the positioning (x values)

### ISSUE THREAD

**Issue by alonep1** [elementor/elementor#7719]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

As title says,
When trying to use absolute positioning on an RTL website, the x values are reversed
here is a vid example: 
https://www.useloom.com/share/44b6696a79df4a11b23e5090c8356821

## Steps to reproduce
make an rtl website and try to use absolute positioning

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v5.6.42-84.2
	PHP Version: 7.0.33
	PHP Max Input Vars: 10240
	PHP Max Post Size: 24M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1.1
	Site URL: https://hairclinic.co.il
	Home URL: https://hairclinic.co.il
	WP Multisite: No
	Max Upload Size: 24 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: he-IL
	Timezone: Asia/Jerusalem
	Debug Mode: Inactive

== Theme ==
	Name: Leader Child
	Version: 1.0.0
	Author: Pojo
	Child Theme: Yes
	Parent Theme Name: Leader
	Parent Theme Version: 5.0.2
	Parent Theme Author: Pojo Team

== User ==
	Role: administrator
	WP Profile lang: he_IL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.1
		Author: Automattic

	Asset CleanUp: Page Speed Booster
		Version: 1.3.2.5
		Author: Gabriel Livan

	Contact Form DB
		Version: 2.10.32
		Author: Michael Simpson

	Duplicate Post
		Version: 3.2.2
		Author: Enrico Battocchi

	Ele Custom Skin
		Version: 1.0.9
		Author: Liviu Duda

	Elementor
		Version: 2.5.11
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.3
		Author: Elementor.com

	Insert Headers and Footers
		Version: 1.4.3
		Author: WPBeginner

	Loginizer
		Version: 1.4.1
		Author: Raj Kothari

	ManageWP - Worker
		Version: 4.7.7
		Author: GoDaddy

	One Click Accessibility
		Version: 2.0.3
		Author: One Click Accessibility

	PageSpeed Ninja
		Version: 0.9.37
		Author: PageSpeed Ninja

	Polylang
		Version: 2.5.2
		Author: Frédéric Demarle

	Polylang Connect for Elementor
		Version: 1.0.0
		Author: David Decker - DECKERWEB

	Powies Uptime Robot
		Version: 0.9.5
		Author: Thomas Ehrhardt

	Really Simple SSL
		Version: 3.1.5
		Author: Rogier Lankhorst, Mark Wolters

	Redirection
		Version: 4.1.1
		Author: John Godley

	Shield Security
		Version: 7.2.3
		Author: One Dollar Plugin

	Smush
		Version: 3.2.0.1
		Author: WPMU DEV

	WP Rocket
		Version: 3.2.5
		Author: WP Media

	Yoast SEO
		Version: 10.1.1
		Author: Team Yoast


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 
		Author: ManageWP


== Log ==
	: 
Log: showing 20 of 392019-03-28 09:53:20 [info] אלמנטור פרו/Upgrades - _v_2_5_0_woocommerce_menu_cart Start  
2019-03-28 09:53:20 [info] אלמנטור פרו/Upgrades - _v_2_5_0_woocommerce_menu_cart Finished 
2019-03-28 09:53:20 [info] אלמנטור פרו/Upgrades - _v_2_5_0_posts Start  
2019-03-28 09:53:21 [info] אלמנטור פרו/Upgrades - _v_2_5_0_posts Finished 
2019-03-28 09:53:21 [info] אלמנטור פרו/Upgrades - _v_2_5_0_portfolio Start  
2019-03-28 09:53:21 [info] אלמנטור פרו/Upgrades - _v_2_5_0_portfolio Finished 
2019-03-28 09:53:21 [info] אלמנטור פרו/Upgrades - _v_2_5_0_products Start  
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_products Finished 
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_sitemap Start  
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_sitemap Finished 
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_popup_border_radius Start  
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_popup_border_radius Finished 
2019-03-28 09:53:22 [info] אלמנטור פרו/Upgrades - _v_2_5_0_form Start (2 of 5, 40%) 
2019-03-28 09:53:23 [info] אלמנטור פרו/Upgrades - _v_2_5_0_form Start (3 of 5, 60%) 
2019-03-28 09:53:24 [info] אלמנטור פרו/Upgrades - _v_2_5_0_form Start (4 of 5, 80%) 
2019-03-28 09:53:25 [info] אלמנטור פרו/Upgrades - _v_2_5_0_form Start (5 of 5, 100%) 
2019-03-28 09:53:26 [info] אלמנטור פרו/Upgrades - _v_2_5_0_form Finished 
2019-03-28 09:53:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.4.8',
  'to' => '2.5.2',
)]
2019-04-01 08:34:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '2.5.10',
  'to' => '2.5.11',
)]
2019-04-01 08:34:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '2.5.2',
  'to' => '2.5.3',
)]

JS: showing 2 of 2JS: 2019-03-25 08:00:01 [error X 1][https://hairclinic.co.il/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.8:2:15986] Cannot read property \'model\' of undefined 
JS: 2019-04-01 11:46:21 [error X 6][https://hairclinic.co.il/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.11:2:180006] Cannot read property \'errors\' of undefined 

PHP: showing 1 of 1PHP: 2019-03-28 09:53:26 [warning X 1][/home/wwwhairclinicco/public_html/wp-content/plugins/elementor-pro/core/upgrade/upgrades.php::647] in_array() expects parameter 2 to be array, null given [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by alonep1** [comment]
any follow up?

**Comment by sbc640964** [comment]
Why is there no answer?
It bothers me, too, that it is impossible to work with an absolute location in Hebrew

**Comment by arielk** [comment]
Fixed in v2.6.4

Thanks for your report!
